### DISCUSSION TITLE
Alt-Tabbing out of osu! window changes monitor settings

### DISCUSSION THREAD

**Discussion by Maritsu** [ppy/osu#23101] [Q&A] [answered]
### Type

Game behaviour

### Bug description

After alt-tabbing out of osu!lazer, my monitors shifted. After checking it with `xrandr`, there it was - my monitor configuration was changed and my primary display was disabled, causing everything to move to my secondary monitor.

### Screenshots or videos

Excuse the live recording, I'm not sure OBS could handle whatever is going on here. Apologies for the Streamable link, GitHub only allows files up to 10MB.
[See video here](https://streamable.com/fpsds1)

### Version

2023.305.0-lazer, but has happened for any version really

### Logs

[database.log](https://github.com/ppy/osu/files/11129744/database.log)
[input.log](https://github.com/ppy/osu/files/11129745/input.log)
[network.log](https://github.com/ppy/osu/files/11129746/network.log)
[performance.log](https://github.com/ppy/osu/files/11129747/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11129748/runtime.log)

**Comment by peppy** [comment, marked as answer]
We've never heard of this issue from a linux user before, which means it's 99% related to your configuration.
